project-base/storefront/components/Pages/Order/TransportAndPayment/TransportAndPaymentSelect/TransportAndPaymentSelect.tsx

```diff
@@ -55,7 +56,7 @@ export const TransportAndPaymentSelect: FC<TransportAndPaymentSelectProps> = ({
     );
 
     const renderTransportListItem = (
-        transportItem: TypeTransportWithAvailablePaymentsAndStoresFragment,
+        transportItem: TypeTransportWithAvailablePaymentsFragment & Partial<TypeTransportStoresFragment>,
         isActive: boolean,
     ) => (
         <TransportAndPaymentListItem key={transportItem.uuid} isActive={isActive}>
```

Comment:
You can also rewrite this line:

`pickupPlaceDetail={getPickupPlaceDetail(transport, pickupPlace, transportItem)}`

to 

`pickupPlaceDetail={isActive && pickupPlace ? pickupPlace : undefined}`

at least I think so... it would also allow you to remove that function (`getPickupPlaceDetail`)